app/api/schema/users.py

```diff
@@ -69,6 +69,17 @@ class Meta:
     deleted_at = fields.DateTime(dump_only=True)
     details = fields.Str(allow_none=True)
     contact = fields.Str(allow_none=True)
+    contact_name = fields.Str(allow_none=True)
+    phone = fields.Str(allow_none=True)
+    email = fields.Str(allow_none=True)
```

Comment:
what is this? Users already have an email? You are adding a field called email in to the user schema! Users already have an email, that's how we identify them?

Reply:
@CosmicCoder96 Yes, I agree. But I think that sometimes users might prefer to give another email address for their billing information?
But either ways, the names would result in a conflict. Got that. I'll rename them as something like`billing_email`

Reply:
No, the issue says this:
(Add fields for the following information: Contact Name*, Phone, Email (**Use account email here**), Country*, Tax ID, Registered Entity or Company, Business Address*, City*, ZIP Code*, Additional Info)

Reply:
Okay. Will keep it that way. Will be using the account email here.